creator/Documents/CommandBlocks.md

```diff
@@ -25,127 +25,116 @@ In this guide, we will set up a command block chain to look for a certain type o
-- **datavalue** is an extra argument used to handle variations of the same block. For instance, wool has many different colors. If you set it to `wool` or `wool -1`, it will look for all types of wool. If you set it to `wool 1`, it will only look for orange wool.
+- **position** is the coordinates in the world where the command will look for the specific block. Example: `0 4 0`.
+- **tileName** is the type of block you are looking for. Example: `diamond_block`.
+- **dataValue** is an optional argument used to handle variations of the same block. For this tutorial, we won't be needing to use this.
 
-If you are following this tutorial, you will be in a flat world. The flat world has no blocks placed above the height, or y, value of 4. Let’s set the command input to look for any type of wool block near the origin of the map.
+If you are following this tutorial, you will be in a flat world. As you walk around, you can see your player's position in the top-left of the screen. Let's check for a block near the origin.
 
-1. In the Command Input section, input the following: `/testforblock 0 4 0 wool -1`
-1. Close the GUI.
+1. In the Command Input section, input the following: `/testforblock 0 4 0 diamond_block`.
+1. Close the interface.
 
 ## Command Block Chains
 
-Command blocks have multiple types it can be set to. A command block set to a Chain type means that it will execute its' command if the previous command block executed. You can connect multiple types of command blocks and chain them together to execute a series of logic. The next step in the gameplay loop is to check if the player is able to receive a reward.
+For the next step, we need to place a chain command block. As mentioned above, chain command blocks execute when a block pointing into them is activated. Make sure you place the block like the image showing the arrows.
 
 1. Place another command block next to the existing block. Make sure that the new block is placed in front of the direction that the arrow is pointing.
-1. Click the **Use** button on the newly created block to open the command block GUI.
+1. Open the command block interface.
 1. Set the **Block Type** to **Chain**.
 1. Set the **Condition** to **Conditional**.
 1. Set the **Redstone** to **Always Active**.
-1. Set the **Command Input** to  `/scoreboard players test @p wool_placed 0 0`
+1. Set the **Command Input** to  `/testfor @p[tag=!placed_block]`
 
-Since the second block is set to **Conditional**, the block will only execute if the first command block gives a true value. If a player places a wool block in the correct location, the next block will check if the player's Wool_placed score is 0.
+Since the second block is set to **Conditional**, this new block will only execute if the first command block succeeds, meaning the diamond block was present. This command checks to see if the player hasn't already received the reward.
```

Comment:
I changed this because "gives a true value" is a very weird way of expressing this concept, especially when the phrasing "succeeds" is already used previously in the guide.